src/convertToObject.js

```diff
@@ -6,7 +6,32 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arr = sourceString.split(';');
+  const arr2 = [];
+
+  arr.map((e) => {
+    if (e.length > 4) {
+      arr2.push(e.split(':'));
+    }
+  });
+
+  const arrFlat = [];
+
+  arr2.flat().map((e) => {
```

Comment:
```suggestion
  arr2.flat().map((e) => {
```

do you really need to use `flat`?